### PR TITLE
chore: use same prettier setting as Kibana's

### LINKED ISSUE #151: [Other] Use the same prettier config as Kibana
### Task summary

To avoid having a different coding style within the organization(read more about it [here](https://www.smashingmagazine.com/2012/10/why-coding-style-matters/)), I'd like to suggest using the same prettier config as Kibana's config.

It is going to be a simple task but considering there are many PRs opened at the moment, and running lint --fix would potentially cause some conflicts, I'd recommend we do it when the biggest PRs are merged.

### Linked resources

Kibana's prettierrc: https://github.com/elastic/kibana/blob/main/.prettierrc

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by kyungeunni**
closes #151

Copied over `.prettierrc` from Kibana so that we align code style across the team within the organization.

No need to review every single file that has changed, but take a look at `.prettierrc` file.

### CODE REVIEW

File: e2e/services/browser.ts [hunk 1]
```diff
@@ -22,12 +22,12 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
...
 
-import { chromium, Browser } from "playwright";
-import { env } from "../services";
+import { chromium, Browser } from 'playwright';
+import { env } from '../services';
 
 type ConnectRetryParams = { url?: string; timeout?: number; interval?: number };
 
-const sleep = ms => new Promise(resolve => setTimeout(resolve, ms));
+const sleep = (ms) => new Promise((resolve) => setTimeout(resolve, ms));
```
Comment (justinkambic) [inline e2e/services/browser.ts:30]: Personally, I kinda like omitting the parens if the type is implicit, but I'm fine with staying unified with Kibana's `.prettierrc`.

**Review by justinkambic — APPROVED** [review]
LGTM

File: e2e/services/browser.ts (hunk 1 above)
Comment (kyungeunni) [reply e2e/services/browser.ts:30]: Hmm actually I prefer the same 🤔

File: demo-app/components/Ad.js [hunk 2]
```diff
@@ -1,15 +1,10 @@
-import React from "react";
+import React from 'react';
```
Comment (lucasfcosta) [inline demo-app/components/Ad.js:1]: IMO we should avoid linting the `demo-app` itself as that's a separate project we're just bundling in. It seems to me that's unnecessary to lint it given it's only used for testing and it's maintained separately. That would also make linting quicker.

**Review by lucasfcosta — APPROVED** [review]
LGTM but tbh I'd prefer to avoid linting the demo-app if possible given we just bundled it in from its own repo. No big deal though.

File: demo-app/components/Ad.js (hunk 2 above)
Comment (kyungeunni) [reply demo-app/components/Ad.js]: Also updated .eslintrc too

**Review by lucasfcosta — APPROVED** [review]
LGTM now that we're ignoring the `demo-app` 😄

**Comment by kyungeunni** [comment]
closing in favor of #197.
Figured it is better to start from a clean state rather than resolving conflicts.

**Pull request closed without merging**